.gitattributes

```diff
@@ -76,4 +76,5 @@ tests/src/JIT/Performance/CodeQuality/BenchmarksGame/reverse-complement/revcomp-
 tests/src/JIT/Performance/CodeQuality/BenchmarksGame/k-nucleotide/knucleotide-input.txt text eol=lf
 tests/src/JIT/Performance/CodeQuality/BenchmarksGame/k-nucleotide/knucleotide-input-big.txt text eol=lf
 tests/src/performance/Scenario/JitBench/Resources/word2vecnet.patch text eol=lf
+tests/src/CLRTest.Execute.Batch.targets text eol=rclf
```

Comment:
This is a typo, but this doesn't feel like the right fix. MSBuild should write this with all the appropriate encoding.

Reply:
That being said @jkotas there is a change in the MSBuild version being used with this change. Arcade is bumping us from a 2.1 to a 2.2 SDK. Are you aware of any such issues between these version changes?

Reply:
> Are you aware of any such issues between these version changes?

No

Reply:
@hoyosjs Could you investigate the SDK upgrade issue to see if we can get a better fix?

Reply:
I have a strong feeling that is not the thing to blame here now. If anything, I'd have more of an expectation on our use of CDATA and WriteLinesToFile:
https://github.com/dotnet/coreclr/blob/24cdacaeb1e42a92051b32a6c4051b7a12ff2770/tests/src/CLRTest.Execute.Batch.targets#L426-L429
We compose one giant blob, and MSBuild will write it as such. See microsoft/msbuild#1401 and microsoft/msbuild#3445. Need to confirm if it's this.

Reply:
Interesting. FWIW, the line endings in the targets file are now CRLF with my change to the .gitattributes file, but still the .cmd test runners are messed up, and the current R2R CI legs are going to still fail.

Would it be better to write these runners as powershell scripts, and a .cmd that calls the ps script?

Reply:
> Would it be better to write these runners as powershell scripts

It would be great to reduce writing separate scripts for Windows and Linux.